vm/src/stdlib/io.rs

```diff
@@ -8,6 +8,8 @@ cfg_if::cfg_if! {
         type Offset = i64;
     }
 }
+use crate::stdlib::os::errno_err;
```

Comment:
Place a `cfg` here and on the next line to get rid of the warnings for windows.

Reply:
Thanks for catching this!